### PR TITLE
Feat changelog 20220803

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,122 @@
...
+
+  ``ou:technique``
+    Add a form to record specific techniques used to achieve a goal.
+
+  ``ou:technique:taxonomy``
+    Add a form to record analyst defined taxonomy of different techniques.
+
+  ``risk:attack``
+    Add a ``techniques`` secondary property to record techniques employed
+    during attack.
```
Comment (Cisphyx) [inline CHANGELOG.rst:35]: 'the' attack

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,122 @@
...
+
+  ``ou:org``
+    Add a ``techniques`` secondary property to record techniques employed by
+    the org.
+
+  ``ou:technique``
+    Add a form to record specific techniques used to achieve a goal.
+
+  ``ou:technique:taxonomy``
+    Add a form to record analyst defined taxonomy of different techniques.
```
Comment (Cisphyx) [inline CHANGELOG.rst:31]: 'an analyst defined taxonomy'

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,122 @@
...
+
+  ``risk:threat``
+    Add a form to record a threat cluster or subgraph of threat activity to
+    a single group.
+
+- Annotate the following light edges.
+  (`#2778 <https://github.com/vertexproject/synapse/pull/2778>`_)
+
+  ``targets``
+    When used with a ``out:org``, ``ou:campaign``, ``risk:threat``, or
```
Comment (Cisphyx) [inline CHANGELOG.rst:68]: 'out:org'

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,122 @@
...
+      - ``via:ipv6``
+      - ``via:phone``
+
+  ``risk:compromise``
+    Add a ``techniques`` secondary property to record techniques employed
+    during the compromise.
+
+  ``risk:threat``
+    Add a form to record a threat cluster or subgraph of threat activity to
+    a single group.
```
Comment (Cisphyx) [inline CHANGELOG.rst:62]: the 'to a single group' reads kinda weird, seems like an 'attributable' is missing somewhere

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,122 @@
...
+
+- Annotate the following light edges.
+  (`#2778 <https://github.com/vertexproject/synapse/pull/2778>`_)
+
+  ``targets``
+    When used with a ``out:org``, ``ou:campaign``, ``risk:threat``, or
+    ``risk:attack`` nodes, the edge indicates the target node was targeted
```
Comment (Cisphyx) [inline CHANGELOG.rst:69]: nodes is plural but the sentence starts with 'with a'

File: CHANGELOG.rst [hunk 6]
```diff
@@ -4,6 +4,122 @@
...
+    by the source node.
+
+  ``uses``
+    When used with a ``ou:campaign`` or ``risk:attack`` node, the edge
```
Comment (Cisphyx) [inline CHANGELOG.rst:73]: 'an' ou:campaign

File: CHANGELOG.rst [hunk 7]
```diff
@@ -4,6 +4,122 @@
...
+  If the previous behavior is desired, use the ``--yield`` option when
+  invoking the ``count`` command.
+  (`#2779 <https://github.com/vertexproject/synapse/pull/2779>`_)
+- Add ``$lib.random.int()`` API to Storm for generating random integers.
+  (`#2783 <https://github.com/vertexproject/synapse/pull/2783>`_)
+- Add a new tool, ``synapse.tools.livebackup`` for taking a live backup of
+  a service.
+  (`#2788 <https://github.com/vertexproject/synapse/pull/2788>`_)
+- The Storm ``$lib.jsonstor.cacheset()`` API now returns a dict containing the
+  path and time. The ``$lib.jsonstor.cacheget()`` API now has a argument to
```
Comment (Cisphyx) [inline CHANGELOG.rst:86]: 'an' argument

File: CHANGELOG.rst (hunk 4 above)
Comment (vEpiphyte) [reply CHANGELOG.rst:62]: That was taken from the doc property for the form :thinking:

Comment (Cisphyx) [reply CHANGELOG.rst:62]: the doc prop is slightly different: "A threat cluster or subgraph of threat activity attributable to one group."

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**